docker-compose.yml

```diff
@@ -2,11 +2,12 @@
 version: '3.7'
 services:
   pmm-managed-server:
-    build:
-      context: .
-      args:
-        PMM_VERSION: ${PMM_VERSION:-perconalab/pmm-server:dev-latest}
-      dockerfile: devcontainer.Dockerfile
+    image: perconalab/pmm-server:dev-container
+#    build:
+#      context: .
+#      args:
+#        PMM_VERSION: ${PMM_VERSION:-perconalab/pmm-server:dev-latest}
+#      dockerfile: devcontainer.Dockerfile
```

Comment:
Why not to remove it?

Reply:
It is useful then you need to build container locally.